docs/installation.rst

```diff
@@ -182,8 +182,8 @@ users on 10.10 (Yosemite) and above you only need one step:
 If you want to build cryptography yourself or are on an older OS X version
 cryptography requires the presence of a C compiler, development headers, and
 the proper libraries. On OS X much of this is provided by Apple's Xcode
-development tools.  To install the Xcode command line tools open a terminal
-window and run:
+development tools.  To install the Xcode command line tools (on 10.9+) open a
```

Comment:
Please write this as OS X 10.9+ (with pip's increasing version number, it gets sadly ambigious)